dcat/index.html

```diff
@@ -2639,7 +2639,7 @@ <h4>Property: spatial resolution</h4>
           More complex descriptions of various aspects of spatial precision, accuracy, resolution and other statistics can be provided using the Data Quality Vocabulary [[?VOCAB-DQV]]. </p>
 
 	  <div class="note">
-            <p>This specification allows two datatypes (<a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a> and <a data-cite="XMLSCHEMA11-2#double"><code>xsd:double</code></a>) to avoid easy mistakes that would make metadata non-compliant. For example, [[JSON-LD]] converts numbers to <a data-cite="XMLSCHEMA11-2#double"><code>xsd:double</code></a> by default, and properly generating <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a> requires the use of strings with an explicit or coerced datatype. In [[Turtle]], seemingly minor modifications can change the datatype of a value: `100.0` is an <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a>, while `1e2` is an <a data-cite="XMLSCHEMA11-2#double"><code>xsd:double</code></a>.</p>
+            <p>As for the use of datatype, note that [[JSON-LD]] converts numbers to <a data-cite="XMLSCHEMA11-2#double"><code>xsd:double</code></a> by default, and properly generating <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a> requires the use of strings with an explicit or coerced datatype. In [[Turtle]], seemingly minor modifications can change the datatype of a value: `100.0` is an <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a>, while `1e2` is an <a data-cite="XMLSCHEMA11-2#double"><code>xsd:double</code></a>.</p>
```

Comment:
```suggestion
            <p>As for the use of datatype, note that [[JSON-LD]] converts numbers to <a data-cite="XMLSCHEMA11-2#double"><code>xsd:double</code></a> or <a data-cite="XMLSCHEMA11-2#integer"><code>xsd:integer</code></a>, and properly generating <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a> requires the use of strings with an explicit or coerced datatype. In [[Turtle]], seemingly minor modifications can change the datatype of a value: `100.0` is an <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a>, while `1e2` is an <a data-cite="XMLSCHEMA11-2#double"><code>xsd:double</code></a>.</p>
```
Numbers are converted to float or integer depending on their fractional part (note that 100.0 in JSON-LD  ends up as "100"^^xsd:integer, despite the trailing ".0" .
I also suggest to remove "by default" because there is no way to change how *numbers* are converted to literals. The workaround is to not use numbers at all.